avocado/utils/linux_modules.py

```diff
@@ -153,3 +165,31 @@ def module_is_loaded(module_name):
 def get_loaded_modules():
     lsmod_output = process.system_output('/sbin/lsmod').splitlines()[1:]
     return [line.split(None, 1)[0] for line in lsmod_output]
+
+
+def check_kernel_config(config_name):
+    """
+    Use a running kernel config file to  get config information
+    :param  config_name:Name of kernel config to search
+    :type config_name:str
+    :return: Config status in running kernel
+    :rtype: enum
```

Comment:
Well the rtype is integer... But I don't think it's necessary to define it anyway...